src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div>
+          <div class="inside">Surname: 
```

Comment:
Maybe it would be better to style the div above this one? And then I would change ```inside``` class name to something more appropriate :)